department-of-veterans-affairs/caseflow
# Enable TaskTable to use API requests to filter tasks

Resolves #11309

### Description
Builds query string to request filtered tasks from the back end and displays the filtered tasks.

### Acceptance Criteria
- [ ] TaskTable component's pagination, filtering, and sorting use API requests instead of happening on the front-end.

### Testing Plan
1. Ensure pagination is enabled with `FeatureToggle.enable!(:use_task_pages_api)`
2. Change user to any VLJ support member and navigate to /organizations/vlj-support
3. Click the filter button next to any column and select a filter!
4. Ensure the tasks filter correctly!!!
5. Pay special attention to filtering by AOD status as well
